### ISSUE TITLE
Prisma does not work on 32-bit Windows

### ISSUE THREAD

**Issue by livthomas** [prisma/prisma#10860]
### Bug description

When I try to run `prisma migrate dev` on a 32-bit Windows 10, I get the following error:
```
Prisma schema loaded from prisma\schema.prisma
Error: Get config: Error: Command failed with UNKNOWN: C:\Users\livor\Projects\livthomas\pawnshop-next\node_modules\@prisma\engines\query-engine-windows.exe cli get-config
spawn UNKNOWN
```
When I try to use Prisma client inside my application on the same system, I get this error:

![Screenshot from 2021-12-24 12-13-18](https://user-images.githubusercontent.com/2910853/147360680-146e4e1d-2a90-4e49-809b-3a0f1b91b4de.png)

### How to reproduce

1. Create a new Prisma project on Windows 10 (32-bit)
2. Set engine type to `binary`
3. Try to run `prisma migrate dev`

### Expected behavior

According to [this comment](https://github.com/prisma/prisma/issues/7660#issuecomment-993583492), Prisma should work with 32-bit Node.js when you use `binary` engine type.

### Prisma information

```prisma
generator client {
  provider   = "prisma-client-js"
  engineType = "binary"
}

datasource db {
  provider = "sqlite"
  url      = "file:../storage.db"
}
```

### Environment & setup

- OS: Windows 10 (32-bit)
- Database: SQLite
- Node.js version: v16.13.1

### Prisma Version

```
3.6.0
```

**Comment by aqrln** [comment]
I don't think it is possible to run 64-bit binaries on a 32-bit operating system.

The comment you linked to is about using Prisma with 32-bit _Node.js_, which is possible with `binary` engine type since Node.js and the Query Engine will be separate processes in that case, but it still requires 64-bit Windows and 64-bit CPU to run the Query Engine in the first place.

**Comment by aqrln** [comment]
Changed the issue type to `kind/feature` since it's not a bug but a request to support another platform.

**Comment by livthomas** [comment]
@aqrln So Prisma doesn't run on 32-bit operating systems, right? This is super-unclear since there is no documentation page describing which specific platforms are supported and which aren't. Maybe you could put together something like that.

**Comment by aqrln** [comment]
> So Prisma doesn't run on 32-bit operating systems, right?

We don't provide pre-built binaries for 32-bit operating systems. Currently it will only run if you build the engines from sources.

> This is super-unclear since there is no documentation page describing which specific platforms are supported and which aren't.
That's a very good point, I don't think I've seen anything like that in our docs. https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#binarytargets-options

**Comment by aqrln** [comment]
Sorry, mis-clicked the wrong button.
